### PR TITLE
Add support install binary windows

### PR DESCRIPTION
**Pull request by davesnx**
Fixes https://github.com/sanack/node-jq/pull/54 with Windows when trying to install the binary.

### CODE REVIEW

File: scripts/install-binary.js [hunk 1]
```diff
@@ -28,23 +29,26 @@ if (jqExists()) {
   process.exit(0)
 }
 
-const build = new BinBuild()
-  .src(`${JQ_INFO.url}/${JQ_INFO.version}/${JQ_INFO.version}.tar.gz`)
+if (process.platform === 'win32' || process.platform === 'win64') {
```
Comment (davesnx) [inline scripts/install-binary.js]: Are you sure around that? I never touch cross-platform scripts on Node, but seems incorrect.
Could you please add some info there or try to use some tested lib to do that?

**Comment by davesnx** [comment]
Like the approach 👍

File: scripts/install-binary.js (hunk 1 above)
Comment (davesnx) [inline scripts/install-binary.js]: Seems fragile this part to me, but if it works: Yes, okay fine!

File: src/command.js [hunk 2]
```diff
@@ -2,6 +2,9 @@ import path from 'path'
 import { parseOptions } from './options'
 
 let JQ_PATH = path.join(__dirname, '../bin/jq')
```
Comment (davesnx) [inline src/command.js]: could you separate this into `path.join(__dirname, '..', 'bin', 'jq')`.
That affects the paths on windows

File: scripts/install-binary.js [hunk 3]
```diff
@@ -34,16 +34,23 @@ if (jqExists()) {
...
 if (process.platform === 'win32' || process.platform === 'win64') {
-  download(`${JQ_INFO.url}/${JQ_INFO.version}/jq-${process.platform}.exe`)
-  .pipe(fs.createWriteStream(`${FILE_PATH}`))
+  download(`${JQ_INFO.url}/${JQ_INFO.version}/jq-${process.platform}.exe`, OUTPUT_DIR)
+  .then(() => {
+    fs.renameSync(path.join(OUTPUT_DIR, `jq-${process.platform}.exe`), FILE_PATH)
+  })
+  .catch((err) => {
+    console.error(err)
+    process.exit(1)
```
Comment (davesnx) [inline scripts/install-binary.js]: Doesn't relaly new!

File: scripts/install-binary.js [hunk 4]
```diff
@@ -14,29 +15,44 @@ const JQ_INFO = {
...
-    return fs.statSync((path.join(OUTPUT_DIR, 'jq'))).isFile()
+    return fs.statSync(FILE_PATH).isFile()
   } catch (err) {
     return false
   }
 }
 
 if (jqExists()) {
-  console.log('jq is already installed')
+  console.log('jq is already installed here:', FILE_PATH)
```
Comment (davesnx) [inline scripts/install-binary.js]: This will cause conflicts in the future, just for keep it in mind.

Comment (mackermans) [inline scripts/install-binary.js]: For global installations you mean?

**Comment by MaanooAk** [comment]
Oups... Just saw this (take look at #58)

**Comment by davesnx** [comment]
Make sense to kill all the install binary here @mackermans and merge it with appveyor?

**Pull request closed without merging**